project/sherLOCKED/Assets/Scripts/Sherry/SherryInteract.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class SherryInteract : MonoBehaviour
+{
+    public Text interact;
+    private bool inCollider = false;
+
+    void Update()
+    {
+        if (inCollider)
+        {
+            if (Input.GetKeyDown("e") || Input.GetKeyDown("space"))
+            {
+                interact.text = "Good Job. You did it!";
+            }
+        }
+    }
+
+    void OnTriggerEnter2D(Collider2D other)
+    {
+        if (other.gameObject.CompareTag("Interactable") && !inCollider)
+        {
+            inCollider = !inCollider;
```

Comment:
Is there a reason this is swapping, rather than explicit setting to `true`? The former seems less clear to me